ansible/roles/atmo-vnc/tasks/realvnc.yml

```diff
@@ -1,7 +1,8 @@
 ---
 
 - name: set environment variables for legacy systems
-  set_fact: ansible_python_interpreter=/usr/bin/python
+  set_fact:
+    ansible_python_interpreter: /usr/bin/python
```

Comment:
It's old code, I wasn't sure if I could trash it. I'm pretty sure it's not necessary, but I'll have to double check that.